src/patternfly/demos/Page/page-template-context-selector.hbs

```diff
@@ -0,0 +1,55 @@
+{{#> menu-toggle menu-toggle--modifier="pf-m-full-width" menu-toggle--IsExpanded=page-template-sidebar--hasContextSelectorExpanded}}
+  {{#> menu-toggle-icon}}
+    <i class="fas fa-fw fa-cogs" aria-hidden="true"></i>
+  {{/menu-toggle-icon}}
+  {{#> menu-toggle-text}}
+    Administrator
+  {{/menu-toggle-text}}
+  {{#> menu-toggle-controls}}
+    {{> menu-toggle-toggle-icon}}
+  {{/menu-toggle-controls}}
+{{/menu-toggle}}
```

Comment:
Just because it's usually easier for maintenance, I'd use this here. And I might also add a param to `menu-toggle.hbs` for `menu-toggle--IsFullWidth` so we can use that in place of `menu-toggle--modifier="pf-m-full-width"`

```suggestion
{{> menu-toggle
  menu-toggle--modifier="pf-m-full-width"
  menu-toggle--IsExpanded=page-template-sidebar--hasContextSelectorExpanded
  menu-toggle--text="Administrator"
  menu-toggle--icon="cogs fa-fw"}}
```